### PR TITLE
RDM-8816: Runtime stubbing of idam user info

### PR DESCRIPTION
**Pull request by mario-paniccia**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/ccd/test/stubs/service/controllers/StubResponseController.java [hunk 1]
```diff
@@ -43,6 +56,7 @@
 public class StubResponseController {
 
     private static final Logger LOG = LoggerFactory.getLogger(StubResponseController.class);
+    static final String WIREMOCK_STUB_MAPPINGS_ENDPOINT = "/__admin/mppings";
```
Comment (danlysiak) [inline src/main/java/uk/gov/hmcts/reform/ccd/test/stubs/service/controllers/StubResponseController.java:59]:
```suggestion
    static final String WIREMOCK_STUB_MAPPINGS_ENDPOINT = "/__admin/mappings";
```

Comment (mario-paniccia) [reply src/main/java/uk/gov/hmcts/reform/ccd/test/stubs/service/controllers/StubResponseController.java:59]: Ah! I did change that last minute to simulate an error and forgot to revert it :) 
Thanks @danlysiak

File: dependency-check-suppressions.xml [hunk 2]
```diff
@@ -52,5 +52,9 @@
     <notes>Temporary suppression for on-going developments</notes>
     <cpe>cpe:2.3:a:checkstyle:checkstyle:*:*:*:*:*:*:*:*</cpe>
     <cve>CVE-2019-10782</cve>
+    <cve>CVE-2020-9488</cve>
+    <cve>CVE-2020-9484</cve>
+    <cve>CVE-2020-5407</cve>
+    <cve>CVE-2020-5408</cve>
```
Comment (mario-paniccia) [inline dependency-check-suppressions.xml:58]: thanks @danlysiak 👍

File: README.md [hunk 3]
```diff
@@ -96,6 +96,27 @@ docker image rm <image-id>
...
+```
+/idam-user
+```
+
+Example:
+
+```
+curl -X POST \
+--header 'content-type: application/json' \
+--data '{"email":"auto.test.cnp@gmail.com","roles":["caseworker-autotest1","caseworker-autotest2","caseworker-autotest3","caseworker","ccd-import"],"sub":"auto.test.cnp@gmail.com","uid":"7689","name":"CCD Auto Test (Stub2)","given_name":"CCD","family_name":"Auto Test (Stub2)"}' \
```
Comment (smathangi) [inline README.md:114]: Why not existing wire-mock API. http://wiremock.org/docs/api/#tag/Stub-Mappings/paths/~1__admin~1mappings/post
(we already have an example userinfo mapping file in the code).

**Pull request merged**